src/schema_formats/XML/GMP.xml.in

```diff
@@ -6091,7 +6096,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
     </ele>
     <ele>
       <name>copy</name>
-      <summary>The UUID of an existing TLS certificate</summary>
+      <summary>The UUID of an existing TLS certificate.</summary>
```

Comment:
Summaries shouldn't have a fullstop.  The generator adds them.

Reply:
I've changed that in 1e59119cdd27e048e8788bba54db7170bbf1276f.
There are other places that have this, but I'd clean them up separately so the changes can be backported more easily.

Reply:
Yes, good.